pillar/caddy/ocw_build.sls

```diff
@@ -0,0 +1,74 @@
+{% set app_name = 'ocw-build' %}
+{% set env_settings = salt.cp.get_url("https://raw.githubusercontent.com/mitodl/salt-ops/main/salt/environment_settings.yml", dest=None)|load_yaml %}
+{% set ENVIRONMENT = salt.grains.get('environment', 'applications-qa') %}
+{% set env_data = env_settings.environments[ENVIRONMENT] %}
```

Comment:
I don't see an `applications-qa` or `applications-production` environment in `environment_settings.yml` yet. Is that OK?